### DISCUSSION TITLE
Template Floorplan

### DISCUSSION THREAD

**Discussion by AllenDBoston** [siliconcompiler/siliconcompiler#956] [Development]
Is siliconcompiler capable of importing a .def Floorplan template? This could be extremely useful for MPW projects.

**Comment by aolofsson** [comment]
Yes, you can use the read parameter to read in a def on a per step basis. In the case of using asicflow and you want to read in a def for floorplanning, the step would be "floorplan". 

https://docs.siliconcompiler.com/en/latest/reference_manual/schema.html#read
